src/Components/RecordingButton/index.jsx

```diff
@@ -16,11 +16,13 @@ export const RecordingButton = (props) => {
         props.setTitle("");
         props.setArtwork("");
         props.setSongNotFound(false);
-        // props.setSpotifyID("")
+        // props.setSpotifyArtistID("")
```

Comment:
remove comments

Reply:
Comments are prep for the spotify stuff we'll be working on this week